pubspec.yaml

```diff
@@ -1,17 +1,37 @@
 name: dynamic_color_theme
 description: Dynamically change your color theme, including light and dark mode. Persist the colors across restarts.
-version: 1.1.2
+version: 1.2.0
 homepage: 'https://github.com/DFreds/dynamic_color_theme'
 
+# The following line prevents the package from being accidentally published to
+# pub.dev using `pub publish`. This is preferred for private packages.
+publish_to: 'none' # Remove this line if you wish to publish to pub.dev
```

Comment:
I'm unfamiliar with this, but it sounds like it shouldn't be here since I'd be publishing it to pub.dev.